packages/tools/devtools/devtools-view/src/WindowMessageRelay.ts

```diff
@@ -25,6 +25,8 @@ import {
  * to communicate with the debugger registry.
  * This ensures that we don't "abuse" the power of local interaction to do things that might not be possible (or need
  * to be done differently) with a message passing mechanism that crosses the boundary of the window.
+ *
+ * @internal
```

Comment:
Why is this needed?

Reply:
It has no functional purpose, but it's something we can do to indicate to consumers of the package "this class is not exposed for you to use directly or subclass it; it's (kind-of) an implementation detail that we need to export to make things work but it's not part of the public API of the package that you can trust".

Reply:
In the future, we will leverage API-Extractor to omit such items from the typings rollup we expose publicly with the package.

Reply:
As for why the change was made: we didn't previously export this particular IMessageRelay implementation, but it is a generally useful one, and is used by the example app.